packages/rebber-plugins/__tests__/__snapshots__/rebber.test.js.snap

```diff
@@ -145,17 +145,19 @@ a code without lang
 `;
 
 exports[`code+caption 1`] = `
-"\\\\begin{codeBlock}[With Source]{python}
+"\\\\begin{CodeBlock}{python}
 print('bla')
-\\\\end{codeBlock}
+\\\\captionof{listing}{With Source}
+\\\\end{CodeBlock}
 
 
 
-\\\\begin{codeBlock}[With Source][1,2]{python}
+\\\\begin{CodeBlock}[1,2]{python}
 print('bla')
 print('bla')
 print('bla')
-\\\\end{codeBlock}"
+\\\\captionof{listing}{With Source}
+\\\\end{CodeBlock}"
```

Comment:
.

Reply:
`\captionof` should be outside, but right after `CodeBlock` :)

Reply:
What about images? Current situation is:

```tex
\begin{center}
\includegraphics{url}
\captionof{figure}{my caption}
\end{center}
```

Reply:
`\captionof` could go outside `center`, but that would not change anything. So the current situation is ok.